app/controllers/katello/application_controller.rb

```diff
@@ -72,14 +73,18 @@ def current_organization
       end
     end
 
-    def current_organization=(org)
+    def current_organization
+      current_organization_data&.name
+    end
```

Comment:
if foreman is providing its on current_organization helper, can we just delete ours?

Reply:
I think we can, I will try it out